features/bevragen/persoon/adressering/adres-regels/in-onderzoek.feature

```diff
@@ -0,0 +1,116 @@
+    | naam                                                       | waarde           |
+    | inOnderzoek.adresregel1                                    | <adresregel1 io> |
+    | inOnderzoek.adresregel2                                    | <adresregel2 io> |
+    | inOnderzoek.adresregel3                                    | <adresregel3 io> |
+    | inOnderzoek.land                                           | <land io>        |
+    | inOnderzoek.datumIngangOnderzoekVerblijfplaats.type        | Datum            |
+    | inOnderzoek.datumIngangOnderzoekVerblijfplaats.datum       | 2002-07-01       |
+    | inOnderzoek.datumIngangOnderzoekVerblijfplaats.langFormaat | 1 juli 2002      |
+
+    Voorbeelden:
+    | aanduiding in onderzoek | adresregel1 io | adresregel2 io | adresregel3 io | land io | type                               |
+    | 080000                  | true           | true           | true           | true    | hele categorie verblijfplaats      |
+    | 080900                  |                | true           |                |         | hele groep gemeente                |
+    | 080910                  |                | true           |                |         | gemeente van inschrijving          |
+    | 081000                  |                |                |                |         | hele groep adreshouding            |
+    | 081010                  |                |                |                |         | functie adres                      |
+    | 081030                  |                |                |                |         | datum aanvang adreshouding         |
+    | 081100                  | true           | true           |                |         | hele groep adres                   |
+    | 081110                  | true           |                |                |         | straatnaam                         |
+    | 081115                  |                |                |                |         | naam openbare ruimte               |
+    | 081120                  | true           |                |                |         | huisnummer                         |
+    | 081130                  | true           |                |                |         | huisletter                         |
+    | 081140                  | true           |                |                |         | huinummertoevoeging                |
+    | 081150                  | true           |                |                |         | aanduiding bij huisnummer          |
+    | 081160                  |                | true           |                |         | postcode                           |
+    | 081170                  |                | true           |                |         | woonplaatsnaam                     |
+    | 081180                  |                |                |                |         | identificatiecode verblijfplaats   |
```

Comment:
Dus in dit geval wordt wel inOnderzoek.datumIngangOnderzoekVerblijfplaats opgenomen, maar is er verder geen enkel veld in onderzoek met waarde true? Ik denk dat dat niet correct is.

Reply:
aangepast